DumbView/format.py

```diff
@@ -58,7 +58,7 @@ def formatAlignedRead2(alnReader, refWindow, aln, useColor=False, realign=False)
     transcript = clippedRead.transcript(orientation="genomic")
     reference = clippedRead.reference(orientation="genomic")
 
-    if realign:
+    if doRealign:
         reference, read, transcript = realign(reference, read, transcript)
 
```

Comment:
I see.  My last change broke things.  Did I forget to run the tests?

Thanks for fixing this.